Makefile

```diff
@@ -94,7 +94,6 @@ coverage: build
 acceptance:
 # The first `stop` stops and cleans up any containers from previous runs.
 	(cd $(RUN) && export COCKROACH_IMAGE="$(COCKROACH_IMAGE)" && \
-	  ../build/build-docker-dev.sh && \
```

Comment:
Removing this line makes running `make acceptance` from my dev box more irritating as I'd first have to do `build/build-docker-dev.sh`. Am I missing something?